.eslintrc.js

```diff
@@ -41,6 +41,14 @@ module.exports = {
             },
             rules: {
                 '@typescript-eslint/no-floating-promises': 'off',
+                '@typescript-eslint/no-unused-vars': [
+                    'error',
+                    {
+                        argsIgnorePattern: '^_',
+                        varsIgnorePattern: '^_',
+                        caughtErrorsIgnorePattern: '^_',
+                    },
+                ],
```

Comment:
This would keep linting, but this is a common way to handle unused defined parameters but the typescript linter does not 

Many linters will treat parameters prefixed with an underscore as "intentionally unused" and won't complain about them.



https://stackoverflow.com/questions/64052318/how-to-disable-warn-about-some-unused-params-but-keep-typescript-eslint-no-un